### PR TITLE
Add short descriptions of Khoury GSA activities

### PR DESCRIPTION
**Pull request by mhyee**
I took the descriptions from an old recruiting email draft.

For now, they're at the very bottom of the single-page website.

We may want to consider splitting the website into multiple pages, as it's starting to get long.

### CODE REVIEW

**Comment by ulysses4ever** [comment]
This looks good to me. Let's merge this one and go from there.

File: index.md [hunk 1]
```diff
@@ -60,3 +61,44 @@ If you are more of an “ask-around” person, do not hesitate to post your ques
...
+
+* _Undistinguished Lecture Series._ An event where a student gives a talk on any
+topic they desire, as long as it is not their research. This was typically held
+at the beginning of a Game Night. A fun end-of-semester tradition was to play
+“slideshow karaoke.”
+
+**Representing Khoury PhD students**
+
+* _Meetings with the Khoury admins._ Approximately once a month, we meet with the
+administration (currently Sarah Gale, Frank Tip, and Bryan Lackaye) to discuss
```
Comment (ulysses4ever) [inline index.md:88]: Actually, we probably shouldn't have put the old names? I think omitting the names is the easiest at the moment.

Comment (mhyee) [reply index.md:88]: Good catch, I forgot to take this out. Agree, it's probably easier to just remove. I'll update directly on `main`.

Comment (ulysses4ever) [reply index.md:88]: Cool!

**Pull request merged**